components/StoreProducts.js

```diff
@@ -4,7 +4,8 @@ import { View } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
 import Product from './Product';
 
-import { Button, Title } from '../styles/shared';
+import {TextButton} from '../components/BaseComponents';
+import { Button ,Title} from '../styles/shared';
```

Comment:
Same as above - let's get rid of anything unused with these changes.

Also, keep in mind that this means this needs to be merged _after_  #25, and the changes here may need to be updated for elements @tommypoa added (minimal IIRC)